lib/bowling.rb

```diff
@@ -0,0 +1,123 @@
+class Bowling
+  attr_reader :round_score, :total_score, :round, :ball, :strike_bonus, :spare_bonus, :bonus_ball, :bonus_score, :last_strike
+
+  def initialize
+    @round_score = 0
+    @total_score = 0
+    @round = 1 
+    @ball = 1
+    @strike_bonus = nil
+    @spare_bonus = nil
+    @bonus_ball = 1
+    @bonus_score = 0
+    @last_strike = nil
+  end
+
+  def score_input
+    input = $stdin.gets
+    scorer(input.to_i)
+  end
+
+  def input_score_message
+    "Please enter your score for round #{round}, ball #{ball}"
+  end
+
+  def scorer(pins)
+    @strike_bonus = nil if @round > 11
+    @spare_bonus = nil if @round > 11
+
+    strike_calculator(pins)
+    spare_calculator(pins)
+
+    if @round >= 10
+      @total_score += pins
+      next_round
+    else
+
+      @round_score += pins
+
+      if strike
+        @strike_bonus = true
+        @last_strike = true
+        add_to_total_score
+        next_round
+      elsif spare
+        @spare_bonus = true
+        add_to_total_score
+        next_round
+        @last_strike = false
```

Comment:
When I cloned and ran your Rspec, the only tests that failed seemed to be related to games with multiple strikes and spares. Looks like you were working on debugging that at the time of the pull request, so keep going with that :)